content/knowledge-codemagic/build-inputs.md

```diff
@@ -10,6 +10,10 @@ Build inputs are customizable parameters you can define within your workflow to
 
 Inputs are workflow-specific and are defined in `codemagic.yaml` under the `inputs` mapping (see the [example](#minimal-example) below). The started workflow receives specified input values in the `inputs` context, i.e. `${{ inputs.inputId }}` is replaced with the value passed to input with identifier `inputId`.
 
+{{<notebox>}}
```

Comment:
perhaps add it to the top under Overview

Reply:
Done, please review @icarusdust